sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/TraceUtil.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp.implementation;
+
+import com.azure.core.amqp.exception.AmqpException;
+import com.azure.core.implementation.tracing.TracerProxy;
+import com.azure.core.util.Context;
+import reactor.core.publisher.Signal;
+
+import java.util.Optional;
+
+public final class TraceUtil {
+
```

Comment:
Add a private no-op constructor here just to defend against people creating instances of this class for no reason.